src/masternodes/mn_checks.cpp

```diff
@@ -1227,36 +1236,164 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
 
     Res operator()(const CMintTokensMessage& obj) const {
         // check auth and increase balance of token's owner
-        for (const auto& kv : obj.balances) {
-            const DCT_ID& tokenId = kv.first;
-
-            if (Params().NetworkIDString() == CBaseChainParams::MAIN && height >= static_cast<uint32_t>(consensus.FortCanningCrunchHeight) &&
```

Comment:
The FortCanningCrunchHeight consensus code to prevent loan tokens from being minted is missing and needs to be restored.